index.md

```diff
@@ -4,35 +4,17 @@ layout: default
 
 {{ site.description }}
 
-## Start Here
-Please bring your own laptop to work on. Just getting started with Ruby, Rails, Python or Node.JS? We'll help you set up your environment so you can get started!
-
-Breakfast snacks like bagels and fruit are provided.
-
-Here's some great links to helpful learning material including our challenges, exercises, tutorials, and important documentation.
+## Welcome
+Please bring your own laptop to work on. We'll help you set up your environment so you can get started! Breakfast snacks like bagels and fruit are provided.
 
 - [New Members: Start Here](/getting_started/)
 - [Track your Challenge Progress](http://challengeprogress.herokuapp.com/)
-- [Challenges](/challenges/)
+- [Challenge Guide](/challenges/)
```

Comment:
I intentionally renamed Challenge Guide to Challenges. It made the URL and navbar titles shorter, and I wanted to keep things consistent. Do you disagree with shortening it to Challenges?